File: lexiString.py
a = "polikujmnhytgbvfredcxwqaz"
b = "abcd"
index = []
for i in b:
    ind = a.index(i)
    index.append(ind)

index = sorted(index)
for i in index:
    print(a[i], end="")
print()

File: lexiString.cpp
#include<iostream>
#include<string>
using namespace std;
int main() {
    string alpha = "polikujmnhytgbvfredcxwqaz";
    string given = "abcd";
    for(int i = 0; i<alpha.length(); i++) {
        if(given.find(alpha[i]) != string::npos) {
            cout<<alpha[i];
        }
    }
}